### PR TITLE
tesseract: add serial-num-pack option

### PR DESCRIPTION
**Pull request by Zarana-Parekh**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Updating formula for Tesseract to include option for installing serial number language pack.

### CODE REVIEW

File: Formula/tesseract.rb [hunk 1]
```diff
@@ -92,6 +98,9 @@ def install
     system "./configure", *args
 
     system "make", "install"
+    if build.with? "serial-num-pack"
+      resource("snum").stage{ mv "snum.traineddata", share/"tessdata" }
```
Comment (ilovezfs) [inline Formula/tesseract.rb]: needs a space after stage

Comment (ilovezfs) [inline Formula/tesseract.rb]: Should this be using `pkgshare`?

**Comment by chrismattmann** [comment]
Great work @Zarana-Parekh thanks for the review @ilovezfs hoping this can get in soon!

File: Formula/tesseract.rb (hunk 1 above)
Comment (DomT4) [inline Formula/tesseract.rb]: It's probably alright; `tessdata` isn't a particularly generic name & it's not placing any _files_ in the top-level prefix.

File: Formula/tesseract.rb [hunk 2]
```diff
@@ -92,6 +98,9 @@ def install
     system "./configure", *args
 
     system "make", "install"
+    if build.with? "serial-num-pack"
+      resource("snum").stage { mv "snum.traineddata", share/"tessdata" }
```
Comment (dunn) [inline Formula/tesseract.rb]: This can just be `(pkgshare/"tessdata").install resource("snum")`.

**Comment by chrismattmann** [comment]
Hi @dunn @Zarana-Parekh can this be merged?

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `tesseract: add serial-num-pack option.`. This format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md). Thanks!

**Comment by chrismattmann** [comment]
thank you @MikeMcQuaid and @Zarana-Parekh !

**Pull request merged**